src/conduits/RwaMultiSwapOutputConduit.sol

```diff
@@ -385,8 +382,9 @@ contract RwaMultiSwapOutputConduit {
     function expectedGemAmt(uint256 wad) public view returns (uint256 amt) {
         require(psm != address(0), "RwaMultiSwapOutputConduit/psm-not-hooked");
 
-        uint256 to18ConversionFactor = 10**_sub(18, GemAbstract(gem()).decimals());
-        return _mul(wad, WAD) / _mul(_add(WAD, PsmAbstract(psm).tout()), to18ConversionFactor);
+        uint256 to18ConversionFactor = 10**(18 - uint256(GemAbstract(gem()).decimals()));
```

Comment:
We should add a comment that we are ensuring `18 - uint256(GemAbstract(gem()).decimals()´ won't overflow before.